src/styles/components/generic/_grid.scss

```
@@ -0,0 +1,71 @@
+$viewports: (
+    sm: 480,
+    md: 780,
+    lg: 1024,
+);
+
+@mixin makeGrid {
```

Stylelint (17.12.0)

[error] Expected mixin name "makeGrid" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)